extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ColumnChunkReaderImpl.java

```diff
@@ -38,7 +38,7 @@ public class ColumnChunkReaderImpl implements ColumnChunkReader {
     private final ColumnChunk columnChunk;
     private final SeekableChannelsProvider channelsProvider;
     private final Path rootPath;
-    private final Compressor decompressor;
+    private final CompressorAdapter decompressor;
```

Comment:
this can never be closed, so never is returned to the pool, as modified in this patch - this block in CodecWrappingCompressorImpl.close() will never execute:
```
            if (innerDecompressor != null && innerDecompressorPooled) {
                CodecPool.returnDecompressor(innerDecompressor);
            }
```

Reply:
We still aren't returning decompressors, right?